certification/internal/shell/scorecard_basic-check-spec-test.go

```diff
@@ -0,0 +1,135 @@
+package shell
+
+import (
+    "os/exec"
+    "os"
+    "fmt"
+    "io/ioutil"
+
+    "github.com/komish/preflight/certification"
+    "github.com/sirupsen/logrus"
+)
+
+type ScorecardBasicCheckSpecTest struct{}
+
+func (p *ScorecardBasicCheckSpecTest) Validate(bundle string, logger *logrus.Logger) (bool, error) {
+    
+    //Curentdir
+    homedir, err := os.Getwd()
+    artifactsdir := homedir+"/artifacts"
+    test := "operator_bundle_scorecard_BasicCheckSpecTest.json"
+    
+    if err != nil {
+        logger.Error("unable to get current directory: ", err)
+        return false, err
+    }
+
+    tmpdir, err := os.MkdirTemp("", "preflight")
+
+    if err != nil {
+        logger.Error("unable to create TempDir: ", err)
+        return false, err
+    }
+    defer os.RemoveAll(tmpdir)
+
+    operatordir := tmpdir+"/test-operator-bundle"
+    bundledir := operatordir+"/bundle"
+    
+    err = os.MkdirAll(bundledir, 0777)
+    if err != nil {
+        logger.Error("unable to create subdirs: ", err)
+        return false, err
+    }
+
+    err = os.Chdir(bundledir)
+    if err != nil {
+        logger.Error("unable to change dir: ", err)
+        return false, err
+    }
+
+    stdouterr, err := exec.Command("oc", "image", "extract", bundle).CombinedOutput()
+    if err != nil {
+        logger.Error("unable to execute oc image extract on the image: ", err)
+        return false, err
+    }
+
+    err = os.Chdir(tmpdir)
```

Comment:
The same comment as above here; can we pass the `tempdir` to ReadFile instead of changing working dir?

Reply:
operator-sdk scorecard expects current dir so this is the only one  I kept for Chdir.